### ISSUE TITLE
Compare with other related works which use DDIM inversion

### ISSUE THREAD

**Issue by silence-tang** [theEricMa/ScaleDreamer#3]
Hi, this is a great work!
You analyze and compare the ways to model the score function of the distribution of rendered images among SDS, CSD, VSD and ASD. So I am curious that what do you think of works that use DDIM inversion + MSE to optimize the 3D representation, such as LucidDreamer, VividDreamer, Score Distillation via Reparametrized DDIM? Will this line of works perform better or worse (speed, quality, VRAM cost, etc) in text-to-3d tasks compared with ASD?
Looking forward to hearing from you. Thanks!

**Comment by theEricMa** [comment]
Thank you for your interest in our work. That's a great question. While experiments are needed to directly compare KL-divergence-based methods with editing-based methods, I have some intuitive insights:

  1. Quality: VSD and our ASD are based on DDPM, while editing-based methods use DDIM. Generally, DDPM is known to provide better generation quality than DDIM, as indicated by metrics like FID scores in image generation. Thus, ASD might produce better quality. Additionally, our 2D toy experiments show that ASD can generate high-quality results similar to VSD, which is not typically demonstrated by DDIM-based methods.
  
  2. Speed: In each iteration, ASD calls the 2D diffusion model three times (conditional, unconditional, and conditional with a shifted timestep). These operations can run in parallel, requiring minimal VRAM since gradients aren't needed in the U-net. Conversely, editing-based methods need sequential calls to the 2D diffusion models—first for inversion and then for editing. For example, ISM by LucidDreamer may require multiple steps to compute the gradient for each iteration, especially early in training. This sequential process introduces latency with each interaction. In prompt-amortized training, where more iterations are needed, this can become significantly time-consuming.

Personally, I'm curious about the performance of these editing-based methods in prompt-amortized training. However, it's a lower priority on my to-do list at the moment. Sorry about that.

**Comment by silence-tang** [comment]
Thanks for your detailed answers!
I focus mainly on prompt-specific training. So can I draw a conclusion that it is quite likely that DDIM inversion based methods will perform worse in quality and training speed than ASD in prompt-specific training?
By the way, is ASD the SOTA among KL-divergence-based methods or there are some other concurrent works?

**Comment by theEricMa** [comment]
It's hard to say. VSD fine-tunes the 2D diffusion prior, and if this fine-tuning works well without degrading its generative capacity, VSD could outperform ASD. However, VSD tends to be unstable.

**Comment by silence-tang** [comment]
My friend, do you know about some recent works (after april) that also improve SDS, VSD? I want to learn from these works and compare among them. Could you please share if you have read these works. Thanks!

**Comment by theEricMa** [comment]
That is very easy to search. Type in "score distillation" in [arxiv](https://arxiv.org/list/cs.CV/recent) you will find all of them

**Comment by silence-tang** [comment]
OK, thanks
